src/moveit_studio_agent_utils/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.16.3)
+project(moveit_studio_agent_utils)
+
+find_package(ament_cmake REQUIRED)
+find_package(moveit_studio_agent_msgs REQUIRED)
+find_package(moveit_studio_behavior_msgs REQUIRED)
+
+set(THIS_PACKAGE_INCLUDE_DEPENDS
```

Comment:
For completeness, let's put a comment here saying these are packages needed to link against the necessary MoveIt Studio ROS interface definitions. Interface being the official ROS 2 term for "messages + services + actions"